### PR TITLE
feat: profile routes

### PR DESCRIPTION
**Pull request by momentiris**
**Ticket:** [#141](https://trello.com/c/wy0um2ee/141-profilvyer-routing)
**Intent:**
Add profile routing layout component.
**How to test (optional):**
/profile
### All of the following commands should pass.

- [x] `npm test`
- [x] `npm run lint`
- [x] `npm run cypress`

### Browser testing

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] IE11
- [ ] Edge

### CODE REVIEW

File: src/components/Layout/ProfileLayout.tsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+  return (
+    <Grid gridGap="0" gridTemplateRows="auto 1fr auto" p="large">
+      <Flex
+        alignItems="center"
+        flexDirection="column"
+        justifyContent="center"
+        position="relative"
+      >
+        <SavePdf>
```
Comment (lessp) [inline src/components/Layout/ProfileLayout.tsx]:
```suggestion
        <SavePdf aria-role="button">
```

File: src/components/Layout/ProfileLayout.tsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+      >
+        <ProfileButton
+          buttonText="Min profil"
+          isActive={currentRoute === '/profil'}
+          route="/profil/"
+        />
+        <ProfileButton
+          buttonText="Tidslinje"
+          isActive={currentRoute === '/profil/tidslinje'}
+          route="/profil/tidslinje/"
```
Comment (lessp) [inline src/components/Layout/ProfileLayout.tsx]: Can we do `route="./tidslinje"`?

File: src/components/Layout/ProfileLayout.tsx [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+`
+
+const ProfileLayout: React.FC<ProfileLayoutProps> = ({
+  children,
+  currentPath,
+}) => {
+  const { data } = useQuery(GET_PROFESSION_AND_CONTACT)
+
+  const [currentRoute, setCurrentRoute] = useState('')
+  useLayoutEffect(() => setCurrentRoute(currentPath.replace(/\/$/, '')), [
```
[the remark this answers is not in the document]
Comment (momentiris) [reply src/components/Layout/ProfileLayout.tsx]: youre right, not needed.

File: src/components/Layout/ProfileLayout.tsx (hunk 1 above)
Comment (momentiris) [reply src/components/Layout/ProfileLayout.tsx]: Changed to button element instead

File: src/components/Layout/ProfileLayout.tsx (hunk 2 above)
Comment (momentiris) [reply src/components/Layout/ProfileLayout.tsx]: yes we can!

**Pull request merged**